my-profile-webapp/src/main/webapp/partials/local-address.html

```diff
@@ -0,0 +1,33 @@
+<link href="css/app.css" rel="stylesheet" type="text/css"/>
+<div class="portlet-frame" style='display: inline-block; width: 100%;'>
+    <portlet-header app-title="My Profile - Local Address" app-image="img/square.jpg"></portlet-header>
+    <div class="portlet-body">
+        <!-- App Navigation if Needed -->
+        <div class="inner-nav-container">
+            <ul class="inner-nav">
+                <li ><a href="#/main">My Profile</a></li>
+                <li class="active"><a href="#/local">Local Address</a></li>
+                <li><a href="#/emergencyContact">Emergency Contacts</a></li>
+            </ul>
+        </div>
+        <div ng-controller="LocalContactInformationController as localContactInfoCtrl">
+            <div class="alert alert-danger" role="alert" ng-if='error && error.length > 0'>{{error}}</div>
```

Comment:
My editor (and GitHub) flags the ampersands here.  Are those okay or do they need to be escaped?